src/components/profile/components/passport/Passport.tsx

```diff
@@ -0,0 +1,107 @@
+/* Import Dependencies */
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import KeycloakService from "app/Keycloak";
+import { Row, Col } from "react-bootstrap";
+
+/* Import Styling */
+import styles from 'components/profile/profile.module.scss';
+
+/* Import Icons */
+import { faEnvelope } from "@fortawesome/free-solid-svg-icons";
+
+/* Import Webroot */
+import ORCIDIcon from 'webroot/logos/ORCIDLogo.png'
+
+/* Import Components */
+import { Button } from "components/elements/customUI/CustomUI";
+
+
+/**
+ * Component that renders the user passport on the profile page
+ */
+const Passport = () => {
+    return (
+        <div>
+            {/* Head */}
+            <Row className={`${styles.profileCardHead}`}>
+                <Col>
+                    <div className="d-flex justify-content-center position-relative h-100 bgc-primary">
+                        <div className={`${styles.profileCardPicture} d-flex justify-content-center align-items-center position-absolute top-0 bgc-secondary br-round`}>
+                            <p className="tc-white fw-lightBold">
+                                {KeycloakService.GetParsedToken()?.given_name[0]}
```

Comment:
I believe this will only provide the last name, is that intentional?

Reply:
The given_name is your name as in 'Soulaine', and yes this is correct.
I use the first letter of your name as a profile picture, like Google does.